app/scripts/background.js

```diff
@@ -44,8 +45,9 @@ const notificationManager = new NotificationManager()
 global.METAMASK_NOTIFIER = notificationManager
 
 // setup sentry error reporting
-const release = platform.getVersion()
-const raven = setupRaven({ release })
+const releaseVersion = platform.getVersion()
+const raven = setupRaven({ releaseVersion })
+assert.equal('test', 'notest')
```

Comment:
You might wanna remove this one (which is probably the one line that's breaking the e2e tests)

Reply:
super weird, totally thought i removed this. thanks for taking a look.